docs/example-config-files/terraform.bitops.config.yaml

```diff
@@ -1,8 +1,9 @@
 terraform:
     cli: 
       backend-config:
-        - KEY1=foo
-        - KEY2=bar
+        KEY1=foo
```

Comment:
why this change?

Reply:
I've updated this to use;
```
        - KEY1=foo
          KEY2=bar
        - KEY3=baz
```

I just wanted to point out that you can use dash or indent for entries

Reply:
each syntax should do something different:
```
backend-config:
  - KEY1=foo
  - KEY2=bar
```
this is a list and is essentially the same as: `backend-config:["KEY1=foo","KEY2=bar"]`

whereas this:
```
backend-config:
  KEY1=foo
  KEY2=bar
```
would probably be invalid syntax.  no dashes implies an object, but object syntax would be:
```
backend-config:
  KEY1: foo
  KEY2: bar
```
which would essentially be: `backend-config: { KEY1: "foo", KEY2: "bar" }`

this syntax:
```
backend-config:
  - KEY1=foo
    KEY2=bar
  - KEY3=baz
```
is also invalid syntax because it's trying to be a list (denoted by the first line's `-`), but the second item doesn't have the dash.  I suspect this would break yaml.

it's _possible_ maybe that this would evaluate to: `backend-config: ["KEY1=fooKEY2=bar", "KEY3=baz"]`, but I doubt it..

I think this should be a list only, like:
```
backend-config:
  - KEY1=foo
  - KEY2=bar
```

Reply:
I will change this, but I did test locally and it worked
It creates the parameters properly using either dash or indent


**config file**
```
terraform:
    cli: 
      backend-config:
        - KEY1=foo
          KEY2=bar
        - KEY3=baz
      init-upgrade: true
    options:
        command: apply
        version: "0.13.2"
        fetch-kubeconfig: false
```

**Output**
```
script_options: [  -backend-config="KEY1=foo" -backend-config="KEY2=bar" -backend-config="KEY3=baz" --upgrade]
```

Reply:
strange that works. I haven't looked at the parsing code, but this is essentially saying:
```
{
  terraform: {
    cli: {
      backend-config: [
          {
              "KEY1=foo",
              "KEY2=bar"
          },{
              "KEY3=baz"
          }
      ]
    }
  }
```